openshift-maven-plugin/plugin/src/main/java/org/eclipse/jkube/maven/plugin/mojo/build/OpenshiftHelmMojo.java

```diff
@@ -25,7 +25,7 @@
 public class OpenshiftHelmMojo extends HelmMojo {
 
   /**
-   * The generated kubernetes YAML file
+   * The generated Kubernetes YAML file
```

Comment:
```suggestion
   * The generated OpenShift YAML file
```

Reply:
Thanks! Implemented in c03626830525443607c4cccd0715228d76d57c2a.